deeprank2/query.py

```diff
@@ -183,19 +184,30 @@ def build(
         Returns:
             :class:`Graph`: The resulting :class:`Graph` object with all the features and targets.
         """
+        # TODO: Should this be an internal method, as it is only called upon by QueryCollection and not by user?
```

Comment:
Could be that a user is interested in building the graph only. I'd keep it as it is for now.

Reply:
OK. removed the TODO in $$$$$